tests/Acceptance/Service/RequestTest.php

```diff
@@ -34,75 +37,176 @@ protected function setUp(): void
         $this->configProvider = ExampleConfiguration::provideDefaultConfig();
     }
 
-    /**
-     * @test
-     * @dataProvider \Answear\LuigisBoxBundle\Tests\DataProvider\RequestDataProvider::forContentUpdate()
-     */
+    #[Test]
+    #[DataProvider('forContentUpdate')]
```

Comment:
why not from `RequestDataProvider`? It is impossible to load from external static method?